src/components/common/CoreAppWrapper/CoreAppWrapper.tsx

```diff
@@ -0,0 +1,30 @@
+import LanguageProvider from 'providers/LanguageProvider'
+import { NetworkProvider } from 'providers/NetworkProvider'
+import ReactQueryProvider from 'providers/ReactQueryProvider'
+import { ThemeProvider } from 'providers/ThemeProvider'
+import { V1UserProvider } from 'providers/v1/UserProvider'
+import React from 'react'
+import { Provider } from 'react-redux'
+import store from 'redux/store'
+
+/**
+ * Contains all the core app providers used by each page.
+ */
+export const CoreAppWrapper: React.FC = ({ children }) => {
```

Comment:
note on directory structure: we're using "shared" rn in other places. any reason to use "common" instead here?

Reply:
Rationale was that we were trying to cut back on usage of `shared` at a root level since everything in components is technically shared now (specific code is now living under the nextjs pages), but don't have any issue changing to shared here either

Reply:
gotcha, cool with me 👌